Documentation/ApiOverview/Seo/PageTitleApi.rst

```diff
@@ -54,7 +58,7 @@ this method you can create your own logic to define the correct title.
 
     use TYPO3\CMS\Core\PageTitle\AbstractPageTitleProvider;
 
-    class MyOwnPageTitleProvider extends AbstractPageTitleProvider
+    final class MyOwnPageTitleProvider extends AbstractPageTitleProvider
```

Comment:
Please use the snippet from _MyOwnPageTitleProvider.php.
Also adjust the namespace to MyVendor\MySitepackage.